src/setting.rs

```diff
@@ -133,6 +133,42 @@ impl SettingData {
     }
 }
 
+/// Payload of a `DEVSETTING.DAT` file (32 bytes).
+#[binrw]
+#[derive(Debug, PartialEq)]
+#[brw(little)]
+pub struct DevSetting {
+    /// Unknown field.
+    unknown1: [u8; 9],
+    /// "Type of the overview Waveform" setting.
+    pub overview_waveform_type: OverviewWaveformType,
+    /// "Waveform color" setting.
+    pub waveform_color: WaveformColor,
+    /// Unknown field.
+    unknown2: u8,
+    /// "Key display format" setting.
+    pub key_display_format: KeyDisplayFormat,
+    /// "Waveform Current Position" setting.
+    pub waveform_current_position: WaveformCurrentPosition,
+    /// Unknown field.
+    #[br(assert(unknown3 == [0x00; 18]))]
+    unknown3: [u8; 18],
```

Comment:
Could you also add asserts for the other unknown fields?

Reply:
https://github.com/Holzhaus/rekordcrate/pull/62/commits/e370cb93854ba5d04e7e6868702435244f45884b